### DISCUSSION TITLE
3.4 log propagation with emissary and Vault sidecar

### DISCUSSION THREAD

**Discussion by prakasha28** [argoproj/argo-workflows#13315] [Q&A] [answered]
We are using argo workflow with v3.4.0(0.18.0 - Chart version). Our kubernetes version is 1.25.1.
For our applications, We have Cronworkflow, Workflowtemplate object.
In Workflowtemplate, We are using annoations to send logs to fluentbit and also Vault is running as side car container.
When we updgrade from 3.3.2 to 3.4.0, We observed the strange issue that logs are not getting visible on elasticsearch/kibana. Previously it was working.
There must be some change w.r.t to emissary which is causing this trouble.
Also when I stop vault as side car container then logs are coming to kibana but it is not possible for us to stop using vault.
I shared you my cronworkflow, workflowtemplate for reference.
Please check on priority because this is bug which is there on all versions after > 3.4

**Comment by agilgur5** [comment, marked as answer]
Duplicate of #13314. Please don't create more than one issue or discussion about the same problem.

> I shared you my cronworkflow, workflowtemplate for reference.

There's nothing provided in this discussion either, although it is included in the issue, but not minimal and using private images 😕
